docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -103,23 +92,36 @@ spec:
 and create it:
 
 ```shell
-$ kubectl create -f my-crontab.yaml
-crontab "my-new-cron-object" created
+kubectl create -f my-crontab.yaml
 ```
 
-You can then manage our `CronTab` objects using kubectl. Note that resource
-names are not case-sensitive when using kubectl:
+You can then manage your CronTab objects using kubectl. For example:
 
 ```shell
-$ kubectl get crontab
+kubectl get crontab
+```
+
+Should print a list like this:
+
+```console
 NAME                 KIND
 my-new-cron-object   CronTab.v1.stable.example.com
 ```
 
-You can also view the raw JSON data. Here you can see that it contains the custom `cronSpec` and `image` fields from the yaml you used to create it:
+Note that resource names are not case-sensitive when using kubectl,
+and you can use either the singular or plural forms defined in the CRD,
+as well as any short names.
 
-```yaml
-$ kubectl get ct -o yaml
+You can also view the raw JSON data:
+
+```shell
+kubectl get ct -o yaml
+```
+
+You should see that it contains the custom `cronSpec` and `image` fields
```

Comment:
oh, it's just that I tend to use the `describe` subcommand to get additional info about a K8s object (as opposed to `get` with the `-o` option). I don't feel super strongly on this though.